src/bookshelf/rebar.config

```diff
@@ -39,7 +39,7 @@
     {opscoderl_wm, ".*",
         {git, "git://github.com/chef/opscoderl_wm.git",               {branch, "master"}}},
     {sqerl, ".*",
-        {git, "git://github.com/chef/sqerl.git",                      {branch, "master"}}},
+        {git, "git://github.com/chef/sqerl.git",                      {branch, "lbaker/envy"}}},
```

Comment:
This might be a broader discussion of when do we want to create forks.
- Would it be for all of the deps?
- Would it be only if we needed to make a change?